lib/gradient/ast_specifier.ex

```diff
@@ -1,10 +1,10 @@
 defmodule Gradient.AstSpecifier do
   @moduledoc """
   Module adds missing location information to the Erlang abstract code produced
-  from Elixir AST. Moreover it can be used to catch some ast pattern and replace 
+  from Elixir AST. Moreover it can be used to catch some ast pattern and replace
```

Comment:
My editor really dislikes the trailing whitespace on these lines. If desired, I can undo these changes to clean up the PR a bit.

Reply:
That's fine, I don't like these either, but they somehow sneaked in :D